network/src/config.rs

```diff
@@ -32,7 +33,7 @@ pub struct Config {
     /// The maximum number of peers permitted to maintain connections with.
     maximum_number_of_connected_peers: u16,
     /// The default bootnodes of the network.
-    pub bootnodes: RwLock<Vec<SocketAddr>>,
```

Comment:
note for the future: I don't think that list is ever modified during runtime

Reply:
just for tests I think